Client/Ecosia/UI/NTP/NTPTooltip.swift

```diff
@@ -113,15 +156,36 @@ final class NTPTooltip: UICollectionReusableView, NotificationThemeable {
         tail.tintColor = UIColor.theme.ecosia.quarternaryBackground
         background.backgroundColor = UIColor.theme.ecosia.quarternaryBackground
         textLabel.textColor = .theme.ecosia.primaryTextInverted
-        closeImage.tintColor = .theme.ecosia.primaryTextInverted
+        closeButton.tintColor = .theme.ecosia.primaryTextInverted
     }
 
     @objc func tapped() {
         delegate?.ntpTooltipTapped(self)
     }
+    
+    @objc private func closeTapped() {
+        delegate?.ntpTooltipCloseTapped(self)
+    }
+    
+    @objc private func linkButtonTapped() {
+        delegate?.ntpTooltipLinkTapped(self)
+    }
 
     override func prepareForReuse() {
         super.prepareForReuse()
         applyTheme()
     }
+    
+    private func updateTailPosition() {
+        switch tailPosition {
+        case .center:
+            tailLeadingConstraint.isActive = false
+            tailCenterConstraint.isActive = true
+        case .leading:
+            tailCenterConstraint.isActive = false
+            tailLeadingConstraint.isActive = true
+        case .trailing:
+            assertionFailure()
```

Comment:
..wondering if it makes sense to have `.trailing` after all. Apparently using it will have no meaningful effect until it gets implemented 🤔 Did you have a dedicated use case in mind for that?

Reply:
Hm interesting, I was pretty sure I only added cases which we actualyl need. I'll remove this!